server/src/graphql/queries/database.ts

```diff
@@ -10,6 +10,9 @@ export const database: QueryResolvers['database'] = async (
     throw new Error('Unauthorized');
   }
 
-  const database = await prisma.database.findOne({ where: { id: databaseId } });
+  const database = await prisma.database.findOne({
```

Comment:
I think this should rather be in a resolver. That way it's not executed if you don't ask for the subfields :)